learning_journal/models/mymodel.py

```diff
@@ -1,18 +1,19 @@
 from sqlalchemy import (
     Column,
-    Index,
     Integer,
     Text,
+    Unicode,
+    DateTime,
 )
 
 from .meta import Base
 
 
-class MyModel(Base):
-    __tablename__ = 'models'
-    id = Column(Integer, primary_key=True)
-    name = Column(Text)
-    value = Column(Integer)
-
+class Entry(Base):
+    """Entry for journal entries."""
 
-Index('my_index', MyModel.name, unique=True, mysql_length=255)
+    __tablename__ = 'entries'
+    id = Column(Integer, primary_key=True)
+    title = Column(Unicode(128), unique=True)
+    body = Column(Text)
```

Comment:
Given that you are setting the title to `Unicode`, i'd suggest that you use `UnicodeText` here as well.  In terms of what goes in  to PostgreSQL, that is the same type, but it means you can treat both your body text and your titles as unicode in Python, which you really want to do.